kordac/KordacExtension.py

```diff
@@ -72,6 +59,7 @@ def extendMarkdown(self, md, md_globals):
                 md.parser.blockprocessors.add(tag_processor[0], tag_processor[1], tag_processor[2])
 
     def reset(self):
```

Comment:
I know this wasn't you but is `reset` really the best name, people might think it resets template and tags etc.